TROGDOR.md

```diff
@@ -123,6 +123,14 @@ ProcessStopFault stops a process by sending it a SIGSTOP signal.  When the fault
 ### NetworkPartitionFault
 NetworkPartitionFault sets up an artificial network partition between one or more sets of nodes.  Currently, this is implemented using iptables.  The iptables rules are set up on the outbound traffic from the affected nodes.  Therefore, the affected nodes should still be reachable from outside the cluster.
 
+External Processes
+========================================
+Trogdor supports running external processes. This is a generic way to run any process in the system via a configurable command - be it a Python program, bash script, docker image, etc.
```

Comment:
Maybe "running arbitrary commands in external processes. This is a generic way to run any configurable command in the Trogdor platform - be it...."?